axum-macros/Cargo.toml

```diff
@@ -16,9 +16,10 @@ proc-macro = true
 
 [dependencies]
 heck = "0.4"
+itertools = "0.10.3"
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = { version = "1.0", features = ["full"] }
+syn = { version = "1.0", features = ["full", "extra-traits", "visit"] }
```

Comment:
I am not sure why I needed to add these features on my local setup, but can debug this later

Reply:
Maybe you ran tests from inside the axum-macros directory? Running tests for a workspace can have different behavior than doing so for the individual packages, unfortunately.